docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/shape/squeeze-1.rst

```diff
@@ -31,6 +31,12 @@ Squeeze
 
 
 **Attributes**: *Squeeze* operation doesn't have attributes.
+* *pytorch_dynamic_rank*
```

Comment:
Not sure if this is a good name. It is needed for Pytorch, but it can be used in a different framework, it just allows Squeeze to have axis with non-1 shape.

Reply:
Any suggestions for the new name?

Reply:
Done